README.md

```diff
@@ -0,0 +1,44 @@
+# Portfolio Sketlon Project Documentation
+
+In this project I created a mobile version of a Portfolio integrating some features:
+- A header with navigation bar and menu including a logo.
+- A Presentation section that provides general informations for the portfolio's owner.
+
+## Built With
+
+- HTML
+- CSS
+- VSCODE
+
+## Live Demo
+
+Not available yet
+
+
+## Authors
+
+👤 Noureddine MEZIANY
+
+- GitHub: [@nmeziany](https://github.com/nmeziany)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/noureddine-meziany/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Please, consider fixing the link to the MIT.md file - it is broken. I know you can fix it easily, then go-go 👍

Reply:
Licence added!

Reply:
- [ ] Please add MIT.md to fix the problem of the link.